queries/janet_simple/highlights.scm

```diff
@@ -0,0 +1,341 @@
+(bool_lit) @boolean
+(nil_lit) @constant.builtin
+(comment) @comment @spell
+
+["'" "~"] @string.escape
+
+["|" "," ";"] @punctuation.special
+
+["{" "@{" "}"
+ "[" "@[" "]"
+ "(" "@(" ")"] @punctuation.bracket
+
+;; >> Symbols
+
+;; General symbol highlighting
+(sym_lit) @variable
+
+;; General function calls
+(par_tup_lit
+ .
+ (sym_lit) @function.call)
+
+(short_fn_lit
+ .
+ (sym_lit) @function.call)
+
+;; Quoted symbols
+(quote_lit
+ (sym_lit) @symbol)
+
+(qq_lit
+ (sym_lit) @symbol)
+
+;; Dynamic variables
+((sym_lit) @variable.builtin
+ (#lua-match? @variable.builtin "^[*].+[*]$"))
+
+;; Operators
+((sym_lit) @operator
+ (#any-of? @operator
+  "%" "*" "+" "-" "/"
+  "<" "<=" "=" "==" ">" ">="))
+
+((sym_lit) @keyword.operator
+ (#any-of? @keyword.operator
+  "not" "not=" "and" "or"))
+
+;; Definition
+((sym_lit) @keyword
+ (#any-of? @keyword
+  "def" "def-" "defdyn" "defglobal" "defmacro" "defmacro-"
+  "var" "var-" "varglobal"))
+
+((sym_lit) @keyword.function
+ (#match? @keyword.function "^(defn|defn-|fn|varfn)$"))
+
+;; Comment
+((sym_lit) @comment
+ (#any-of? @comment "comment"))
+
+;; Conditionals
+((sym_lit) @conditional
+ (#any-of? @conditional
+  "case" "cond"))
+
+((sym_lit) @conditional
+ (#match? @conditional "^if(\\-.*)?$"))
+
+((sym_lit) @conditional
+ (#match? @conditional "^when(\\-.*)?$"))
+
+;; Repeats
+((sym_lit) @repeat
+ (#any-of? @repeat
+  "for" "forever" "forv" "loop" "repeat" "while"))
+
+;; Exception
+((sym_lit) @exception
+ (#any-of? @exception "error" "errorf" "try"))
+
+;; Includes
+((sym_lit) @include
+ (#any-of? @include "import" "require" "use"))
+
+;; Builtin macros
+;;
+;; (each name (all-bindings)
+;;   (when-let [info (dyn (symbol name))]
+;;     (when (info :macro)
+;;       (print name))))
+((sym_lit) @function.macro
+ (#any-of? @function.macro
+  "%=" "*="
+  "++" "+="
+  "--" "-="
+  "->" "->>" "-?>" "-?>>"
+  "/="
+  "and" "as->" "as-macro" "as?->" "assert"
+  "case" "chr" "comment" "compif" "comptime" "compwhen" "cond" "coro"
+  "def-" "default" "defdyn" "defer" "defmacro" "defmacro-"
+  "defn" "defn-"
+  "delay" "doc"
+  "each" "eachk" "eachp"
+  "eachy" ;; XXX: obsolete
+  "edefer"
+  "ev/do-thread" "ev/gather" "ev/spawn" "ev/spawn-thread"
+  "ev/with-deadline"
+  "ffi/defbind"
+  "fiber-fn"
+  "for" "forever" "forv"
+  "generate"
+  "if-let" "if-not" "if-with" "import"
+  "juxt"
+  "label" "let" "loop"
+  "match"
+  "or"
+  "prompt" "protect"
+  "repeat"
+  "seq" "short-fn"
+  "tabseq" "toggle" "tracev" "try"
+  "unless" "use"
+  "var-" "varfn"
+  "when" "when-let" "when-with"
+  "with" "with-dyns" "with-syms" "with-vars"))
+
+;; All builtin functions
+;;
+;; (each name (all-bindings)
+;;   (when-let [info (dyn (symbol name))]
+;;     (when (and (nil? (info :macro))
+;;                (or (function? (info :value))
+;;                    (cfunction? (info :value))))
+;;       (print name))))
+((sym_lit) @function.builtin
+ (#any-of? @function.builtin
+  "%" "*" "+" "-" "/"
+  "<" "<=" "=" ">" ">="
```

Comment:
I'm confused as to why these are here as well, last wins means that `@function.builtin` would win over operator

Reply:
I'd be happy to remove operators.  I only added because of the Clojure file.